### ISSUE TITLE
Canvas is not rendering nested svg which are rotated 

### ISSUE THREAD

**Issue by MUmer01** [canvg/canvg#977]
I have this SVG:

[svg.txt](https://github.com/canvg/canvg/files/4365300/svg.txt)

That is not rendering the nested SVGs properly.
![image](https://user-images.githubusercontent.com/38096900/77250714-8a8cb880-6c6b-11ea-893a-2e3da83c9d5a.png)

**Comment by gabelerner** [comment]
simplified example:
```
<svg xmlns="http://www.w3.org/2000/svg" width="250" height="300" preserveAspectRatio="xMidYMid meet">
    <svg height="150" width="50" x="0" y="0">
        <defs>
            <style>
                .cls-0-0-1584883614872 * {
                    fill: #f8e71c;
                }
            </style>
        </defs>
        <foreignObject class="cls-0-0-1584883614872" width="100%" height="100%">
            <svg xmlns="http://www.w3.org/2000/svg" transform="rotate(180 0 0)" width="50" height="150" viewBox="0 0 34.02 104.88">
                <defs>
                    <style>
                        .cls-1-1584181679251 {
                        }
                    </style>
                </defs>

                <g id="Layer_2" data-name="Layer 2">
                    <g id="Plan_1" data-name="Plan 1">
                        <rect class="cls-1-1584181679251" width="34.02" height="104.88" rx="17.01"> </rect>
                    </g>
                </g>
            </svg>
        </foreignObject>
    </svg>
</svg>
```

removing the `transform="rotate(180 0 0)"` makes them equal. @dangreen - I don't have time to fix things yet but I can review!

**Comment by dangreen** [comment]
Fixed in [v3.0.7](https://github.com/canvg/canvg/releases/tag/v3.0.7)
